tox.ini

```diff
@@ -39,31 +33,31 @@ basepython = python3.7
 skip_install = true
 deps =
     bandit==1.7.0
-    black==20.8b1
-    flake8==3.8.4
-    isort==5.7.0
-    pydocstyle==5.1.1
+    black==21.5b1
+    flake8==3.9.2
+    flake8-bugbear==21.4.3
+    flake8-comprehensions==3.5.0
+    flake8-docstrings==1.6.0
+    isort==5.8.0
     mypy==0.812
-    pandas-stubs==1.0.4.5
+    pandas-stubs==1.1.0.6
     sqlalchemy-stubs==0.4
 commands =
-    flake8 {[tox]source} tests benchmarks
-    isort --honor-noqa --profile black --check setup.py {[tox]source} tests benchmarks
-    pydocstyle {[tox]source}
-    black -q --check setup.py {[tox]source} tests benchmarks
-    bandit -qr {[tox]source}
-    mypy --config-file .mypy.ini  setup.py {[tox]source} tests
-    mypy --config-file .mypy.ini  {[tox]source} benchmarks
+    flake8 {posargs:{[tox]sources}}
+    isort --check {posargs:{[tox]sources}}
+    black --check {posargs:{[tox]sources}}
+    bandit -qr --exclude "benchmarks/,tests/" {posargs:{[tox]sources}}
```

Comment:
I thought about it, but I'm a bit hesitant. When everything passes, black only prints 2 additional lines:
```text
lint run-test: commands[2] | black --check setup.py src/bluesearch tests benchmarks
All done! ✨ 🍰 ✨
68 files would be left unchanged.
```
but bandit spams the stdout with a huge chunk of text, even if there's not issues:
```text
lint run-test: commands[3] | bandit -r --exclude benchmarks/,tests/ setup.py src/bluesearch tests benchmarks
[main]  INFO    profile include tests: None
[main]  INFO    profile exclude tests: None
[main]  INFO    cli include tests: None
[main]  INFO    cli exclude tests: None
[main]  INFO    running on Python 3.7.9
[node_visitor]  INFO    Unable to find qualified name for module: setup.py
Run started:2021-05-21 10:00:23.943386

Test results:
        No issues identified.

Code scanned:
        Total lines of code: 7646
        Total lines skipped (#nosec): 7

Run metrics:
        Total issues (by severity):
                Undefined: 0.0
                Low: 0.0
                Medium: 0.0
                High: 0.0
        Total issues (by confidence):
                Undefined: 0.0
                Low: 0.0
                Medium: 0.0
                High: 0.0
Files skipped (0):
```
so I decided to keep it silenced, especially because we don't really have to fix bandit-related stuff very often.

What do you think?

Reply:
I see. I am OK with the current implementation then.